src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+@use '../_variables/variables' as *;
+
+.card {
+  width: $CardWidth;
```

Comment:
The variable `$CardWidth` should be defined in the `variables.scss` file. Ensure that it is correctly defined and imported.